helm/pattern-2/README.md

```diff
@@ -0,0 +1,102 @@
+# Helm Charts for deployment of WSO2 API Manager with a separate Gateway and a separate Key Manager
+
+## Prerequisites
+
+* In order to use WSO2 Helm resources, you need an active WSO2 subscription. If you do not possess an active WSO2
+  subscription already, you can sign up for a WSO2 Free Trial Subscription from [here](https://wso2.com/free-trial-subscription).<br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the 
+steps provided in the following quick start guide.<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). This can
+ be easily done via 
+  ```
+  helm install stable/nginx-ingress --name nginx-wso2apim-gw-km-with-analytics --set rbac.create=true
+  ```
+## Quick Start Guide
+>In the context of this document, <br>
+>* `KUBERNETES_HOME` will refer to a local copy of the [`wso2/kubernetes-apim`](https://github.com/wso2/kubernetes-apim/)
+Git repository. <br>
+>* `HELM_HOME` will refer to `<KUBERNETES_HOME>/helm/pattern-2`. <br>
+
+##### 1. Checkout Kubernetes Resources for WSO2 API Manager Git repository:
+
+```
+git clone https://github.com/wso2/kubernetes-apim.git
+```
+
+##### 2. Provide configurations:
+
+1. The default product configurations are available at `<HELM_HOME>/apim-gw-km-with-analytics-conf/confs` folder. Change the 
+configurations as necessary.
+
+2. Open the `<HELM_HOME>/apim-gw-km-with-analytics-conf/values.yaml` and provide the following values.
+
+    `username`: Username of your WSO2 subscription credentials<br>
```

Comment:
@chirangaalwis - please fix the wording here during the unifying work

Reply:
+1 will do.